cloudify_rollback_workflow/workflows.py

```diff
@@ -0,0 +1,207 @@
+# Copyright (c) 2016-2019 Cloudify Platform Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from cloudify.decorators import workflow
+from cloudify.workflows.tasks_graph import make_or_get_graph
+from cloudify.plugins.lifecycle import set_send_node_event_on_error_handler
+
+
+@workflow(resumable=True)
+def start(ctx, operation_parms, run_by_dependency_order, type_names, node_ids,
+          node_instance_ids, ignore_failure, **kwargs):
+    execute_operation(ctx, 'cloudify.interfaces.lifecycle.start',
+                      operation_parms, True, run_by_dependency_order,
+                      type_names, node_ids, node_instance_ids, ignore_failure,
+                      **kwargs)
+
+
+@workflow(resumable=True)
+def stop(ctx, operation_parms, run_by_dependency_order, type_names, node_ids,
+         node_instance_ids, ignore_failure, **kwargs):
+    execute_operation(ctx, 'cloudify.interfaces.lifecycle.stop',
+                      operation_parms, True, run_by_dependency_order,
+                      type_names, node_ids, node_instance_ids, ignore_failure,
+                      **kwargs)
+
+@workflow(resumable=True)
+def restart(ctx, stop_parms, start_parms, run_by_dependency_order, type_names,
+            node_ids, node_instance_ids, ignore_failure, **kwargs):
+    stop(ctx, stop_parms, run_by_dependency_order, type_names,
+         node_ids, node_instance_ids, ignore_failure, **kwargs)
+    start(ctx, start_parms, run_by_dependency_order, type_names,
```

Comment:
I don`t know if its good or not but as i think of it, if user wants to run restart with ignore failure he wants the stop will ignore_failure only.
i did what i think its the best but i can either:
 split it to` ignore_failure_start` and `ignore_failure_restart` parameters.
 or just pass `ignore_failure` to both start and stop.
 What is your opinion?